recipes-ni/initscripts-nilrt/initscripts-nilrt_1.0.bb

```diff
@@ -81,6 +83,7 @@ do_install () {
 	update-rc.d -r ${D} nisetbootmode         start 80 S . stop 0 0 6 .
 	update-rc.d -r ${D} firewall              start 39 S .
 	update-rc.d -r ${D} mountdebugfs          start 82 S .
+	update-rc.d -r ${D} nisetembeddeduixml    start 20 5 .
```

Comment:
Are you missing [the stop orders](https://dev.azure.com/ni/DevCentral/_git/ni-central?path=%2Fsrc%2Frtos%2Frtoslegacyd%2FNIOE_Dunfell_x64_temp%2Fx64_dkms_dunfell_nilrt.mak&version=GC885fcbb1263c692781eb9e1f42250da94e6480a3&line=281&lineEnd=282&lineStartColumn=1&lineEndColumn=1&lineStyle=plain&_a=contents) here?

Reply:
The script does nothing in stop, so its unnecessary right?

Reply:
Hmm... yeah; you're right. Weird that we had a stop sequence set. Maybe the script did something on shutdown at some point in the past.

Reply:
Looks like it was that way [from the beginning](https://review-board.natinst.com/r/227548/) - probably a copy paste issue.